gen/go/proto/v1/config.proto

```diff
@@ -0,0 +1,183 @@
+syntax = "proto3";
```

Comment:
where do we maintain original proto?

Reply:
original proto is here: https://github.com/Traceableai/agent-config/blob/main/proto/ai/traceable/agent/config/v1/config.proto